python-frontend/src/test/java/org/sonar/python/types/InferredTypesTest.java

```diff
@@ -201,6 +201,19 @@ public void test_union_type_annotations() {
       .containsExactlyInAnyOrder("bool");
   }
 
+  @Test
+  public void test_annotated_type_annotation() {
+    TypeAnnotation typeAnnotation = typeAnnotation(
+      "from typing import Annotated",
+      "x : Annotated[str, y]"
+    );
+    assertThat(fromTypeshedTypeAnnotation(typeAnnotation)).isEqualTo(STR);
+    InferredType declaredType = fromTypeAnnotation(typeAnnotation);
+    assertThat(declaredType).isInstanceOf(DeclaredType.class);
+    assertThat(((DeclaredType) declaredType).alternativeTypeSymbols()).extracting(Symbol::fullyQualifiedName)
```

Comment:
_minor:_ I think we can also simplify to `assertThat(((DeclaredType) declaredType)).isEqualTo(DECL_STR);`

Reply:
I didn't know about DECL_STR.
So I can replace several lines with:
```
assertThat(fromTypeAnnotation(typeAnnotation)).isEqualTo(DECL_STR);
```